docs/kubernetes/fsgroup.md

```diff
@@ -0,0 +1,95 @@
+# CSI driver FsGroup User Guide
+
+>**Attention:** 'CSIVolumeFSGroupPolicy' is a Kubernetes feature which is Beta is 1.20+, Alpha(1.19).
+>**Attention:** CSIDriver object 'fsGroupPolicy' field is added in Kubernetes 1.19 and cannot be set when using an older Kubernetes release. This workaround is applicable for 1.19+ k8s versions. For 1.20+ k8s versions the feature will be enabled by default, but the workaround be needed until [issue](https://github.com/kubernetes-sigs/gcp-filestore-csi-driver/issues/77) is resolved.
+
+K8s feature ‘CSIVolumeFSGroupPolicy’  is an alpha feature in K8s 1.19 by which CSI drivers can explicitly declare support for fsgroup.
+Until the feature is beta, Kubernetes only applies fsgroup to CSI volumes that are RWO (ReadWriteOnce). Kubernetes uses fsGroup to change permissions and ownership of the volume to match user requested fsGroup in the pod's SecurityPolicy. As a workaround until the CSIVolumeFSGroupPolicy feature is beta, we can deploy a PV backing a filestore instance in RWO mode, apply the FSGroup and then recreate a PV with RWM (ReadWriteMany) mode, so that it can be used for multi reader write workloads. This workaround does not need any root permissions needed for pods. To read more about CSIVolumeFSGroupPolicy read [here](https://kubernetes-csi.github.io/docs/csi-driver-object.html)
```

Comment:
For the sentence "This workaround does not need...", maybe it would be more clear to write "This workaround does not require pods to run containers as the root user."

Reply:
Ah I gobbled the sentences here. Will fix it